src/components/UI/JSONSchemaForm/BaseInputTemplate/index.tsx

```diff
@@ -55,17 +53,18 @@ const BaseInputTemplate: React.FC<WidgetProps> = ({
       onBlur={handleBlur}
     />
   ) : (
-    <TextInput
+    <TextArea
```

Comment:
I think we still need to support basic input field for some cases. It can be rendered for `number` fields (when type is `number` in schema). I guess it also can be rendered for some other input types, like `email`, `password`, etc. 

Maybe we can render `textarea` only when `inputProps.type === 'text'`?